scss/components/_activities-calendar.scss

```diff
@@ -0,0 +1,180 @@
+$activities-calendar-active-day-background: #cde1ed;
```

Comment:
Please add all the colors on the main variables file for now to eliminate duplicity